backend.py

```diff
@@ -679,3 +761,17 @@ def get_direction_from_coordinates(start_coordinates, stop_coordinates):
     for direction in list(Direction):
         if direction.coor_delta == delta:
             return direction
+
+
+def cards_as_dict(cards):
```

Comment:
přejmenovala bych to `dealt_cards_as_dict`

Reply:
pokud to tedy dává ty rozdané, protože bych spíš řekla že to je balíček karet ne? Proč je potom v return "dealt_cards"? Tak tam bude asi chyba.

Reply:
Není to chyba, je to chaotický prototyp ;). Je to funkce, která vezme seznam karet a vrátí slovník, který se dá poslat před websocket. Tato funkce musí ještě projít změnami a najít své místo ve struktuře backendu. A samozřejmě musíme vědět, kdy a komu se posílají jaké karty, aby ta úvodní zpráva seděla s faktickým stavem.